### DISCUSSION TITLE
💡💡💡 droppable guideline preview option

### DISCUSSION THREAD

**Discussion by addlistener** [GrapesJS/grapesjs#5862] [Ideas]
When you specify a custom rule for droppable, it's hard for a user to tell where it could be dropped. An option to show all the places may help.

<img width="893" alt="image" src="https://github.com/GrapesJS/grapesjs/assets/4531670/e25dbf6d-0406-435e-8577-6c2575e3026b">

**Comment by artf** [comment]
I'd delegate this to [Canvas Spots](https://github.com/GrapesJS/grapesjs/issues/3083#issuecomment-1741728197)

**Reply by addlistener** [reply]
Thank you!
